### PR TITLE
Move common widgets from Game to Mods.Common.

### PR DESCRIPTION
**Pull request by pchote**
These no longer need to live in the engine code.

### CODE REVIEW

File: OpenRA.Mods.Common/OpenRA.Mods.Common.csproj [hunk 1]
```diff
@@ -572,6 +572,32 @@
...
+    <Compile Include="Widgets\ProgressBarWidget.cs" />
+    <Compile Include="Widgets\RGBASpriteWidget.cs" />
+    <Compile Include="Widgets\ScrollItemWidget.cs" />
+    <Compile Include="Widgets\ScrollPanelWidget.cs" />
+    <Compile Include="Widgets\SliderWidget.cs" />
+    <Compile Include="Widgets\SpriteWidget.cs" />
+    <Compile Include="Widgets\TextFieldWidget.cs" />
+    <Compile Include="Widgets\TooltipContainerWidget.cs" />
+    <Compile Include="Widgets\ViewportControllerWidget.cs" />
+    <Compile Include="Widgets\VqaPlayerWidget.cs" />
```
Comment (Mailaender) [inline OpenRA.Mods.Common/OpenRA.Mods.Common.csproj]: Sadly MonoDevelop does not automatically sort those alphabetically. I did it manually in https://github.com/OpenRA/OpenRA/pull/7564.

File: OpenRA.Mods.Common/OpenRA.Mods.Common.csproj [hunk 2]
```diff
@@ -572,6 +572,32 @@
     <Compile Include="UtilityCommands\CheckCodeStyle.cs" />
     <Compile Include="UtilityCommands\ReplayMetadataCommand.cs" />
     <Compile Include="Widgets\Logic\ReplayUtils.cs" />
+    <Compile Include="Widgets\BackgroundWidget.cs" />
```
Comment (Mailaender) [inline OpenRA.Mods.Common/OpenRA.Mods.Common.csproj]: Sadly MonoDevelop does not automatically sort those alphabetically. I did it manually in https://github.com/OpenRA/OpenRA/pull/7564.

**Comment by Mailaender** [comment]
:+1: Removals and additions add up. No regressions in-game.

**Comment by obrakmann** [comment]
looks good to me, :+1:

Needs a rebase.

**Comment by pchote** [comment]
Rebased.

**Comment by obrakmann** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/_compare/2bd53a02671b23485ac212828cb946eb82111519%5E...2bd53a02671b23485ac212828cb946eb82111519)

**Pull request merged**